readme.md

```diff
@@ -39,100 +39,105 @@ The template bakes in a pretty basic webpack config, with no transpiling. To set
 
 1. Install Babel packages and respective loader for webpack.
 
-	``` sh
```

Comment:
It looks like the extra indentation was not necessary, so prettier dropped it. The output is unchanged: https://github.com/notlmn/browser-extension-template/blob/163a5b91683a7897c980629450994ba3248db77e/readme.md

Reply:
Well, actually it looks like it replaced the initial tab with 4 spaces… I'm reverting this part